src/main/java/gololang/FunctionReference.java

```diff
@@ -49,7 +49,11 @@ public FunctionReference(MethodHandle handle, String[] parameterNames) {
       throw new IllegalArgumentException("A method handle cannot be null");
     }
     this.handle = handle;
-    this.parameterNames = parameterNames;
+    if (parameterNames != null) {
```

Comment:
If I recall correctly, findbugs was coplaining that we expose the internal of the object by taking or returning a mutable structure, breaking encapsulation. So I copied it :)
Maybe we can in that case simply mute the error